webots/protos/SensorFusionTrack.proto

```diff
@@ -13,7 +13,7 @@ PROTO SensorFusionTrack [
     floorAppearance PBRAppearance {
       baseColorMap ImageTexture {
         url [
-          "./SensorFusionTrack.png"
+          "./track.png"
```

Comment:
Is this correct? Are you using the normal Track?

Reply:
yes, I use the regular linefollower track.